__tests__/screens/FeedScreen/Content.test.js

```diff
@@ -86,14 +87,57 @@ describe('Content component should run correctly', () => {
     post_performance_comments_score: 1,
     privacy: 'Public'
   };
+  const onPress = jest.fn();
+  const onNewPollFetched = jest.fn();
+  const onPressDomain = jest.fn();
+  const images_url = ['https://detik.jpg'];
 
   afterEach(cleanup);
+  it('renders BlurredLayer with isVisible true when isBlurredPost is true', () => {
+    const mockItem = {
+      ...item,
+      isBlurredPost: true // Set isBlurredPost to true for testing
+    };
+
+    const {getByTestId} = render(
+      <Content
+        message={'halo test'}
+        item={mockItem}
+        images_url={images_url}
+        onNewPollFetched={onNewPollFetched}
+        onPressDomain={onPressDomain}
+        onPress={onPress}
+        topics={item.topics}
+      />
+    );
+
+    const blurredLayer = getByTestId('blurredLayer');
+    expect(blurredLayer.props.isVisible).toBe(true);
+  });
+
+  it('renders BlurredLayer with isVisible false when isBlurredPost is false', () => {
+    const mockItem = {
+      ...item,
+      isBlurredPost: false // Set isBlurredPost to true for testing
+    };
+
+    const {getByTestId} = render(
+      <Content
+        message={'halo test'}
+        item={mockItem}
+        images_url={images_url}
+        onNewPollFetched={onNewPollFetched}
+        onPressDomain={onPressDomain}
+        onPress={onPress}
+        topics={item.topics}
+      />
+    );
+
+    const blurredLayer = getByTestId('blurredLayer');
+    expect(blurredLayer.props.isVisible).toBe(false);
+  });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test cases have been added to check if the `BlurredLayer` component's visibility changes based on the `isBlurredPost` prop. However, there's a minor mistake in the comment at line 121. It should indicate that `isBlurredPost` is set to false for testing, not true.
```diff
-       isBlurredPost: false // Set isBlurredPost to true for testing
+       isBlurredPost: false // Set isBlurredPost to false for testing
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->